synapseclient/models/table_components.py

```diff
@@ -92,7 +129,7 @@ def to_synapse_request(self):
             "quoteCharacter": self.quote_character,
             "escapeCharacter": self.escape_character,
             "lineEnd": self.line_end,
-            "isFirstLineHeader": self.is_file_line_header,
```

Comment:
This is a typo in the original `CsvTableDescriptor` class. It should be `is_first_line_header`